fashion-messaging-app-api/routes/users.js

```diff
@@ -21,17 +19,14 @@ try{
     }
     const hashedPassword = await bcrypt.hash(password, 10);
     const newUser = await User.create({ username, password: hashedPassword, email });
-    req.session.user = newUser;
+    req.session.user = newUser.dataValues;
```

Comment:
I changed this to data values because I noticed while I was debugging that the session was storing some metadata that weren't useful. I only need to store the username, password and email and when I made research, I found it was better not not to store the extra Sequelize data as the code will be more prone to bugs if I do